image/cli/mascli/functions/pipeline_config_workload_scale

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+function find_workload_scale_configs_in_dir(){
+  WORKLOAD_SCALE_PROFILE=Custom
+  configs_found=false
+
+  if [[ ! -d "$WORKLOAD_SCALE_DIR_LOCAL" ]]; then
+    echo_warning "Error: Directory does not exist: $WORKLOAD_SCALE_DIR_LOCAL"
+    exit 1
+  fi
+
+  # Check for valid configuration files within the specified directory
+  if [[ -e "${WORKLOAD_SCALE_DIR_LOCAL}/ibm-mas.yml" ]]; then
+    MAS_WORKLOAD_SCALE_FILE="/workspace/pod-templates/ibm-mas.yml"
+    configs_found=true
+  fi
+
+  if [[ -e "${WORKLOAD_SCALE_DIR_LOCAL}/ibm-sls.yml" ]]; then
+    SLS_WORKLOAD_SCALE_FILE="/workspace/pod-templates/ibm-sls.yml"
+    configs_found=true
+  fi
+
+  # If no valid configuration files were found exit
+  if [ "$configs_found" == false ]; then
+    echo_warning "Error: No valid workload scaling configurations found"
+    exit 1
+  fi
+}
+
+function set_workload_scale_envs() {
+  WORKLOAD_SCALE_DIR_NAME=guaranteed
+  [ "$WORKLOAD_SCALE_PROFILE" == "BestEffort" ] && WORKLOAD_SCALE_DIR_NAME=best-effort
+  WORKLOAD_SCALE_DIR_LOCAL="/mascli/templates/pod-templates/${WORKLOAD_SCALE_DIR_NAME}"
+
+  MAS_WORKLOAD_SCALE_FILE="/workspace/pod-templates/ibm-mas.yml"
+  SLS_WORKLOAD_SCALE_FILE="/workspace/pod-templates/ibm-sls.yml"
+}
+
+function unset_workload_scale_envs() {
+  unset WORKLOAD_SCALE_DIR_LOCAL
+  unset MAS_WORKLOAD_SCALE_FILE
+  unset SLS_WORKLOAD_SCALE_FILE
+}
+
+function config_pipeline_workload_scale() {
+  # Change echo_h3 depending on where we call this func
+  echo_h3 "Configure Scaling Profile"
+  echo "The CLI supports three profiles out of the box, and allows you to define your own custom profile as well."
+  echo "Make a selection from the list below. For more information about the profiles, see https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/"
```

Comment:
@durera is `https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/` the link you had in mind for this?